pkg/watcher/block_test.go

```diff
@@ -137,9 +137,11 @@ func TestBlockWatcher(t *testing.T) {
 		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.ValidatedBlocks))
 		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.MissedBlocks))
 		assert.Equal(t, 1, testutil.CollectAndCount(blockWatcher.metrics.SoloMissedBlocks))
+		assert.Equal(t, 0, testutil.CollectAndCount(blockWatcher.metrics.consecutive))
```

Comment:
one last thing to fix :) 
```suggestion
		assert.Equal(t, 0, testutil.CollectAndCount(blockWatcher.metrics.ConsecutiveMissedBlocks))
```